relay-sampling/src/evaluation.rs

```diff
@@ -16,18 +19,12 @@ use relay_protocol::Getter;
 #[cfg(feature = "redis")]
 use relay_redis::AsyncRedisClient;
 use serde::Serialize;
-use uuid::Uuid;
-
-use crate::config::{RuleId, SamplingRule, SamplingValue};
-#[cfg(feature = "redis")]
-use crate::redis_sampling::{self, ReservoirRuleKey};
 
 /// Generates a pseudo random number by seeding the generator with the given id.
 ///
 /// The return is deterministic, always generates the same number from the same id.
-fn pseudo_random_from_uuid(id: Uuid) -> f64 {
-    let big_seed = id.as_u128();
-    let mut generator = Pcg32::new((big_seed >> 64) as u64, big_seed as u64);
+fn pseudo_random_from_seed(seed: u128) -> f64 {
```

Comment:
Decided to just remove our dependence from the `Uuid` and simply use `u128`.

Reply:
This can stay as uuid now, since the trace id just wraps one now? That should make sure we don't accidentally mess up the seed

Reply:
Yeah, I was thinking of keeping it since it makes the whole thing depend on a much simpler piece of code, but if you feel like reducing the surface for possible errors, I can revert.